pkgs/top-level/perl-packages.nix

```
@@ -10400,6 +10400,9 @@ let
     };
     propagatedBuildInputs = [ ArchiveZip DBFile FileWhich IOString ImageSize JSONXS LWP ParseRecDescent TextUnidecode URI XMLLibXML XMLLibXSLT shortenPerlShebang ];
     doCheck = false;  # epub test fails
```

Instead of disabling all tests, we can remove the failing one.

```suggestion
     preCheck = ''
       rm t/931_epub.t # epub test fails
     ''
```